go.mod

```diff
@@ -37,169 +30,300 @@ require (
 )
 
 require (
-	github.com/accuknox/auto-policy-discovery/src v0.0.0-20230412181753-bb7037ee5965
+	github.com/accuknox/auto-policy-discovery/src v0.0.0-20230427074403-5d03ee9f3358
 	github.com/cavaliergopher/grab/v3 v3.0.1
-	github.com/charmbracelet/bubbles v0.14.0
-	github.com/charmbracelet/bubbletea v0.23.1
-	github.com/charmbracelet/lipgloss v0.6.0
-	github.com/deckarep/golang-set/v2 v2.1.0
-	github.com/evertras/bubble-table v0.14.6
+	github.com/charmbracelet/bubbles v0.15.0
+	github.com/charmbracelet/bubbletea v0.23.2
+	github.com/charmbracelet/lipgloss v0.7.1
+	github.com/deckarep/golang-set/v2 v2.3.0
+	github.com/evertras/bubble-table v0.15.2
 	github.com/google/go-cmp v0.5.9
 	github.com/google/go-github v17.0.0+incompatible
 	github.com/kubearmor/KVMService/src/types v0.0.0-20220714130113-b0eba8c9ff34
 	github.com/kubearmor/KubeArmor/KubeArmor v0.0.0-20221130063629-a2dd0301b858
-	github.com/kubearmor/KubeArmor/deployments v0.0.0-20230307164000-b087f0a8bf7f
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20230307075632-73ce4fa43639
-	github.com/kyverno/kyverno v1.6.10
+	github.com/kubearmor/KubeArmor/deployments v0.0.0-20230426155201-4a0d0af2a5d6
+	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20230426155201-4a0d0af2a5d6
+	github.com/kyverno/kyverno v1.9.2
 	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
-	github.com/onsi/ginkgo/v2 v2.5.1
-	github.com/onsi/gomega v1.24.1
-	k8s.io/api v0.26.0
-	k8s.io/apiextensions-apiserver v0.26.0
-	k8s.io/apimachinery v0.26.0
-	k8s.io/cli-runtime v0.23.2
-	k8s.io/client-go v0.26.0
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
+	github.com/onsi/ginkgo/v2 v2.9.1
+	github.com/onsi/gomega v1.27.4
+	k8s.io/api v0.27.1
+	k8s.io/apiextensions-apiserver v0.27.1
+	k8s.io/apimachinery v0.27.1
+	k8s.io/cli-runtime v0.27.1
+	k8s.io/client-go v0.27.1
```

Comment:
can we use v0.26.4 for these k8s.io packages here, we're facing some issues with pkg `controller-runtime` coexist with v0.27 in KubeArmor. thanks

Reply:
`v0.26.4` can't be used because of breaking changes introduced in the `kyaml` module. `0.27.x` solves this issue (https://github.com/kubernetes/kubernetes/pull/116598) which can't be used as it is incompatible with the operator SDK. Hence we will remain at `v0.26.0` for `k8s.io/...`

Reply:
can you verify this?, bcoz i'm able to build and test with v0.26.4.

Reply:
Can you share your exact dependencies? The ones you are building with.

Reply:
```
k8s.io/api => k8s.io/api v0.26.4
k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.4
k8s.io/apimachinery => k8s.io/apimachinery v0.26.4
k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.4
k8s.io/client-go => k8s.io/client-go v0.26.4
```

i also upgraded the kubearmor dependencies as
```
github.com/kubearmor/KubeArmor/KubeArmor v0.0.0-20230512165341-12b8fd29f1aa
github.com/kubearmor/KubeArmor/deployments v0.0.0-20230510133055-4e30a28b6352
github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20230512165341-12b8fd29f1aa
```

Reply:
`replace` is not meant for production, especially not like it is used above. I was able to use `v0.26.4` by not upgrading `sigs.k8s.io/kustomize/api` and `sigs.k8s.io/kustomize/kyaml`.

> i also upgraded the kubearmor dependencies as

That version has a problem that @DelusionalOptimist mentions in https://github.com/kubearmor/kubearmor-client/pull/312#discussion_r1193364394. Using a version greater than that is not possible as k8s.io dependencies are upgraded in https://github.com/kubearmor/KubeArmor/commit/103aea47afe7def28c9750ad835c1bd633f97c65 which we can't use due to our need to using `v0.26.4`.

Reply:
Folks, I think we'll need to update at least `github.com/kubearmor/KubeArmor/deployments` now or later because with kubearmor v0.10 visibility will depend on configuration stored in ConfigMap and the default values have been updated recently. If we should not be using replace this way, what other workarounds do we have? :thinking: